src/image/ExposureFitsReader.cc

```diff
@@ -223,12 +413,9 @@ class ExposureFitsReader::ArchiveReader {
 
         // Not safe to call getAll if a component cannot be unpersisted
         // Instead, look for the archives registered in the metadata
```

Comment:
Given how you reworked this with `readComponent`, is this comment still necessary?

Reply:
I think so? If I could call `getAll`, I wouldn't need to manually build up a Map of results.